pandas/tests/generic/test_generic.py

```diff
@@ -441,3 +441,11 @@ def test_flags_identity(self, frame_or_series):
         assert obj.flags is obj.flags
         obj2 = obj.copy()
         assert obj2.flags is not obj.flags
+
+    def test_bool_dep(self) -> None:
+        msg = (
```

Comment:
Can you add `# GH#51749` as the first line of any added test

Reply:
done

Reply:
I'm not seeing the comment.